conan/internal/runner/ssh.py

```diff
@@ -1,116 +1,93 @@
+import argparse
 from pathlib import Path
 import pathlib
 import tempfile
 
+from conan.api.conan_api import ConanAPI
 from conan.api.output import Color, ConanOutput
 from conan.errors import ConanException
 
 import os
 from io import BytesIO
 import sys
 
-def ssh_info(msg, error=False):
-    fg=Color.BRIGHT_MAGENTA
-    if error:
-        fg=Color.BRIGHT_RED
-    ConanOutput().status('\n┌'+'─'*(2+len(msg))+'┐', fg=fg)
-    ConanOutput().status(f'| {msg} |', fg=fg)
-    ConanOutput().status('└'+'─'*(2+len(msg))+'┘\n', fg=fg)
+from conan import conan_version
+from conan.tools.scm import Version
+from conans.model.profile import Profile
 
 class SSHRunner:
-
-    def __init__(self, conan_api, command, host_profile, build_profile, args, raw_args):
-        from paramiko.config import SSHConfig
-        from paramiko.client import SSHClient
+    def __init__(
```

Comment:
Avoid formatting and typing changes, it makes difficult to read the changelog and check if some new argument was added to the constructor